blocks/shared-styles/index.story.jsx

```diff
@@ -29,7 +29,7 @@ export const primaryFontBlock = () => {
   );
 };
 
-export const secondaryFontBlock = () => {
+export const secondaryFont = () => {
```

Comment:
cool, looks slightly unrelated but no worries. good improvement if deliberate

Reply:
I changed this to avoid the use of the word block in the storybook side bar, to avoid further confusion with the word block